Classes/Controller/OaiPmhController.php

```diff
@@ -302,7 +303,19 @@ public function mainAction(): ResponseInterface
         $this->view->assign('parameters', $this->parameters);
         $this->view->assign('error', $this->error);
 
-        return $this->htmlResponse();
+        // Generate the XML output.
+        $xmlOutput = $this->view->render();
+
+        // Format the XML.
+        $dom = new \DOMDocument();
```

Comment:
```suggestion
        $dom = new DOMDocument();
```

Reply:
Codacy complained because you declared `use DOMDocument;` at the top of the file, but then added the namespace in this line anyways. The use-statement already makes clear that there is no such function/object in current namespace and the global version should be used.
The reason why `getMetsData()` uses `new \DOMDocument();` is, that before your changes we didn't have an use-statement for `DOMDocument`. Strictly speaking we should remove the backslash from `getMetsData` as well, now.